docs/_posts/2016-11-17-new-for-4.0.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title:  "New for 4.0"
+date:   2016-11-17 15:19:00
+---
+
+TSLint [4.0][0] has been released! With this release comes a few exciting [changes][1]. Some of the highlights:
+* **Fixers** - Do you dread turning on a new rule because of all of the new errors? For some of the most common issues, we'll fix them for you. To use this feature, run `tslint` with the `--fix` option. Here are the rules we fix right now:
```

Comment:
can you also write about & provide examples of how to implement a fixer (perhaps suggest that it's easy) to encourage contributions?